### ISSUE TITLE
meson has a problem with ratbag's configure

### ISSUE THREAD

**Issue by jengelh** [libratbag/libratbag#425]
```
[    3s] /usr/bin/meson --buildtype=plain --prefix=/usr --libdir=/usr/lib64 --libexecdir=/usr/lib --bindir=/usr/bin --sbindir=/usr/sbin --includedir=/usr/include --datadir=/usr/share --mandir=/usr/share/man --infodir=/usr/share/info --localedir=/usr/share/locale --sysconfdir=/etc --localstatedir=/var --sharedstatedir=/var/lib --wrap-mode=nodownload . build -Denable-documentation=false --includedir=/usr/include/libratbag
[    3s] The Meson build system
[    3s] Version: 0.43.0
[    3s] Source dir: /home/abuild/rpmbuild/BUILD/libratbag-0.9.900
[    3s] Build dir: /home/abuild/rpmbuild/BUILD/libratbag-0.9.900/build
[    3s] Build type: native build
[    3s] 
[    3s] Meson encountered an error in file meson_options.txt, line 1, column 0:
[    3s] Invalid kwargs for option "udev-dir": "default"
[    3s] 
[    3s] A full log can be found at /home/abuild/rpmbuild/BUILD/libratbag-0.9.900/build/meson-logs/meson-log.txt
[    3s] error: Bad exit status from /var/tmp/rpm-tmp.HnSBM8 (%build)
[    3s] 
[    3s] 
[    3s] RPM build errors:
[    3s]     Bad exit status from /var/tmp/rpm-tmp.HnSBM8 (%build)
[    3s] 
[    3s] a4 failed "build libratbag.spec" at Sat Nov 11 13:31:06 UTC 2017.
[    3s] 
```

**Comment by jengelh** [comment]
```
abuild@a4:~/rpmbuild/BUILD/libratbag-0.9.900> cat build/meson-logs/meson-log.txt 
Build started at 2017-11-21T16:09:32.496451
Python binary: /usr/bin/python3
Python system: Linux
The Meson build system
Version: 0.43.0
Source dir: /home/abuild/rpmbuild/BUILD/libratbag-0.9.900
Build dir: /home/abuild/rpmbuild/BUILD/libratbag-0.9.900/build
Build type: native build
```

**Comment by whot** [comment]
already fixed with  c8929b50fa, please update to git master, thanks
